src/main/java/school/lemon/changerequest/java/banking/TestBA.java

```diff
@@ -0,0 +1,28 @@
+package school.lemon.changerequest.java.banking;
+
+/**
+ * Created by lbrdev on 10.01.2017.
+ * Project: exceptions.hw1
+ */
+public class TestBA {
+    public static void main(String[] args) {
+        BankAccountImpl bankAccount1 = new BankAccountImpl();
+
+        bankAccount1.balance = 15;
```

Comment:
Please always use getters/setters. Direct access to fields is really bad practice.